jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Chip.java

```diff
@@ -15,7 +19,8 @@
  * To see an example of Chip web element please visit https://vuetifyjs.com/en/components/chips/
  */
 
-public class Chip extends UIBaseElement<ChipAssert> implements HasClick, HasIcon, HasLabel {
+public class Chip extends UIBaseElement<ChipAssert> implements HasClick, HasLabel, HasColor, HasIcon, HasImage,
```

Comment:
As I understand we can create a custom content for the chip. In this case we should be able to create a custom class to describe the content and get ability to get this content as UIElement or cast it to specific class. 

Ask me directly if you have any questions or need an example

Reply:
Let's have a close look in to this!